README.md

```diff
@@ -23,3 +23,7 @@ chmod +x borg-exporter
 sudo mv ./borg-exporter /usr/local/bin/
 sudo borg-exporter enable-systemd
 ```
+
+## Alerting rules
+
+Alerting rules can be found [here](./borg-mixin/prometheus-alerts.yaml).
```

Comment:
Maybe mention to tweak the default settings (backupAlertThreshold), you'll need to generate them by yourself.